Server/mods/deathmatch/logic/CResourceHTMLItem.cpp

```diff
@@ -75,6 +75,45 @@ ResponseCode CResourceHTMLItem::Request(HttpRequest* ipoHttpRequest, HttpRespons
             headers.PushString((*iter).second.c_str());
         }
 
+        std::string sMethod;
```

Comment:
```suggestion
        const char* sMethod = "INVALID";
```
Is the heap allocation inside `std::string` really necessary, if you only assign strings from static storage?
With the suggestion you can also simplify the `default` case blow.
`CLuaArguments::PushString` should support pointer to const char.